### ISSUE TITLE
What is the default credentials for admin?

### ISSUE THREAD

**Issue by Warchant** [sourcegraph/sourcegraph#230]
#### Question description

I just deployed `sourcegraph/server:2.11.2` using docker https://about.sourcegraph.com/docs

But what is the default login and password for admin? Can't get this information anywhere.
If possible, put it next to the docker command.

**Comment by sqs** [comment]
It asks you to create the initial admin username and password the first time time you access it. Did you start it and not see that screen?

**Comment by HeinrichHartmann** [comment]
I saw that prompt for email username and password and got logged in fine.

**Comment by Warchant** [comment]
Oh, I finally figured it out.


I executed `docker run --publish 7080:7080 --rm --volume ~/.sourcegraph/config:/etc/sourcegraph --volume ~/.sourcegraph/data:/var/opt/sourcegraph --volume /var/run/docker.sock:/var/run/docker.sock sourcegraph/server:2.11.2`. Note that it mounts config from ~/.sourcegraph

For some reason I already had this dir (I can not say when it was created). 

After I removed it and re-run sourcegraph docker command, it worked as expected - asked for new admin account. 

Closing this.
